test/e2e_new/dls_test.go

```diff
@@ -61,12 +63,21 @@ func SendsEventWithRetries() *feature.Feature {
 	deadLetterSinkName := feature.MakeRandomK8sName("dls")
 	triggerName := feature.MakeRandomK8sName("trigger")
 	brokerName := feature.MakeRandomK8sName("broker")
+	configName := feature.MakeRandomK8sName("config")
 
 	ev := cetest.FullEvent()
 
+	f.Setup("create broker config", brokerconfigmap.Install(
+		configName,
+		brokerconfigmap.WithBootstrapServer("my-cluster-kafka-bootstrap.kafka:9092"),
+		brokerconfigmap.WithNumPartitions(1),
+		brokerconfigmap.WithReplicationFactor(1),
+	))
```

Comment:
Can we use the constant we have?

Also, can we extract this setup into a separate function to avoid duplication?